subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/InstantExecutionException.kt

```diff
@@ -70,7 +72,7 @@ open class InstantExecutionProblemsException : InstantExecutionException {
         htmlReportFile: File
     ) : this(
         "Instant execution problems found in this build.\n" +
-            "Failing because -D${SystemProperties.failOnProblems} is 'true'.",
+            "This behavior can be changed via --no-${ConfigurationCacheFailOnProblemsOption.LONG_OPTION}.",
```

Comment:
:x: I think _this behavior_ is not clear enough without the _failing because_ part because Gradle will always try to find and report problems, it's the failing bit that can be controlled. How about something like, _Gradle can be made to ignore these sort of problems via --ignore-configuration-cache-problems_?